js/client/modules/@arangodb/testsuites/aql.js

```diff
@@ -39,14 +39,21 @@ const optionsDocumentation = [
 const _ = require('lodash');
 const tu = require('@arangodb/test-utils');
 
+const testPaths = {
+  'shell_client': [ 'js/common/tests/shell', 'js/client/tests/http', 'js/client/tests/shell' ],
+  'shell_server': [ 'js/common/tests/shell', 'js/server/tests/shell' ],
+  'shell_server_only': [ 'js/server/tests/shell' ],
+  'shell_server_aql': [ 'js/server/tests/aql' ]
+};
+
 // //////////////////////////////////////////////////////////////////////////////
 // / @brief TEST: shell_client
 // //////////////////////////////////////////////////////////////////////////////
 
 function shellClient (options) {
-  let testCases = tu.scanTestPath('js/common/tests/shell');
-  testCases = testCases.concat(tu.scanTestPath('js/client/tests/http'));
-  testCases = testCases.concat(tu.scanTestPath('js/client/tests/shell'));
+  let testCases = tu.scanTestPath(testPaths.shell_client[0]);
```

Comment:
I do not see the advantage to fixed path here! It is just obfuscated. Do you want to loop over testPaths.shell_client instead and add all? That would make it more generic.  This applies to all suites.